### PR TITLE
Dtsbps 355 -

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-355

### Change description ###
Merging Terraform changes from master

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: infrastructure/.terraform-version [hunk 1]
```diff
@@ -1 +1 @@
-0.13.3
+0.12.20
```
Comment (mdayican) [inline infrastructure/.terraform-version:1]: ???

File: infrastructure/main.tf [hunk 2]
```diff
@@ -32,8 +32,8 @@ module "db-v11" {
 }
 
 module "staging-db" {
```
Comment (mdayican) [inline infrastructure/main.tf:34]: why do you need staging db ?

File: infrastructure/ithc.tfvars [hunk 3]
```diff
@@ -1 +1,4 @@
 managed_identity_object_id = "65ad7733-26bb-45cf-a14e-6e43d4da330a"
+vault_section        = "preprod"
```
Comment (mdayican) [inline infrastructure/ithc.tfvars:2]: do you really need this?

**Review by mdayican — APPROVED** [review]

**Comment by SomeshRA** [comment]
Closing this as Jenkins is not building the project

**Pull request closed without merging**